pom.xml

```diff
@@ -186,6 +186,14 @@
         <module>testing/trino-tests</module>
     </modules>
 
+    <repositories>
+        <repository>
+            <id>apache-snapshot</id>
+            <name>apache snapshot</name>
+            <url>https://repository.apache.org/content/repositories/orgapacheiceberg-1114</url>
```

Comment:
I see some content at https://repo.maven.apache.org/maven2/org/apache/iceberg/iceberg-core/1.1.0/, so you probably can delete this now.